test/cookbooks/runit_test/templates/default/sv-supervisor_group-log-run.erb

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec svlogd -tt /var/log/yerba-matte
```

Comment:
Reusing the existing log directory of another service is causing integration tests to fail.